### PR TITLE
Added udacity.com to change-password-URLs.json

### PR DESCRIPTION
**Pull request by dwivediabhimanyu**
### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for change-password-URLs.json
- [x] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
- [x] The URL either makes the experience better or no worse than being directed to just the domain in a non-logged-in state

### CODE REVIEW

File: quirks/change-password-URLs.json [hunk 1]
```diff
@@ -91,5 +92,5 @@
     "yahoo.com": "https://login.yahoo.com/account/change-password",
     "zhihu.com": "https://www.zhihu.com/settings/account",
     "zocdoc.com": "https://www.zocdoc.com/patient/editprofile?section=Password",
-    "zoom.us": "https://zoom.us/profile#pwd-form"
+    "zoom.us": "https://zoom.us/profile#pwd-form",
```
Comment (bradcush) [inline quirks/change-password-URLs.json:95]: A trailing comma is not valid `JSON` unfortunately, would you mind removing it please?

Comment (dwivediabhimanyu) [reply quirks/change-password-URLs.json]: Sure! Done.

**Comment by rmondello** [comment]
Hi @dwivediabhimanyu,

We recently made a change to the project to clarify the source of contributions to the project. That involved us changing the pull request template to have this item:

- [ ] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)

Can you confirm that you agree to this?

**Comment by dwivediabhimanyu** [comment]
@rmondello 

- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)

**Pull request merged**